qucs/qucs/Makefile.am

```diff
@@ -42,7 +52,7 @@ MOCHEADERS = \
 
 MOCFILES = $(MOCHEADERS:.h=.moc.cpp)
 
-qucs_SOURCES = node.cpp element.cpp qucsdoc.cpp wire.cpp mouseactions.cpp   \
+qucs_real_SOURCES = node.cpp element.cpp qucsdoc.cpp wire.cpp mouseactions.cpp   \
```

Comment:
"qucs", the executable is a wrapper script managed by autotools.

this PR introduces a custom wrapper script "qucs", generated from "qucs.in", it only lives in the builddir. here it executes "qucs_real".

the "_real" suffix for the autotools wrapper is arbitrary, it is stripped in line 43 (during install).